### PR TITLE
Update polaion ID CEPH-83572684 to test case

### PR DESCRIPTION
**Pull request by msainiRedhat**
Signed-off-by: Manisha Saini <msaini@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/nautilus/rgw/tier-2_rgw_s3tests.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ tests:
       desc: test deploying of cluster setup using ceph-ansible
       destroy-cluster: false
       abort-on-fail: true
-      polarion-id: CEPH-83571467
+      polarion-id: CEPH-83571467, CEPH-83572684
```
Comment (Ameenasuhani) [inline suites/nautilus/rgw/tier-2_rgw_s3tests.yaml:37]: It is recommended to have a single polarion id for direct mapping as it helps when TFA is involved.

**Review by Ameenasuhani — COMMENTED** [review]
It is recommended to have a single polarion id for direct mapping as this helps when TFA is involved.

File: suites/nautilus/rgw/tier-2_rgw_s3tests.yaml (hunk 1 above)
Comment (msainiRedhat) [reply suites/nautilus/rgw/tier-2_rgw_s3tests.yaml:37]: Hi Ameensa,

As we need to add the polarion ID to test scripts if already automated, what can be done here?
This testcase applies to 2 polarion cases...Any suggestions?

Comment (Ameenasuhani) [reply suites/nautilus/rgw/tier-2_rgw_s3tests.yaml:37]: We could check if CEPH-83571467 is covered else where then update a single polarion id here

Comment (msainiRedhat) [reply suites/nautilus/rgw/tier-2_rgw_s3tests.yaml]: I have remove CEPH-83571467 as this case was covered/updated in many other test scripts

**Comment by Ameenasuhani** [comment]
@msainiRedhat Can you please resolve DCO.

**Comment by msainiRedhat** [comment]
@Ameenasuhani Resolved.Could you please review now?

**Review by Ameenasuhani — APPROVED** [review]
LGTM

**Review by vdas-redhat — APPROVED** [review]

**Review by psathyan — APPROVED** [review]

**Pull request merged**